torch/_inductor/compile_fx.py

```diff
@@ -736,14 +737,15 @@ def compile_fx(
             )
 
     if config.cpp_wrapper:
-        # CudaWrapperCodeGen relies on kernel name to find the autotuned cubin file
         with config.patch(
             {
                 "cpp_wrapper": False,
-                "triton.unique_kernel_names": True,
                 "triton.autotune_cublasLt": False,
+                "triton.cudagraphs": False,
+                # CudaWrapperCodeGen relies on kernel name to find the autotuned cubin file
+                "triton.unique_kernel_names": True,
             }
-        ):
+        ), V.set_real_inputs(example_inputs_):
```

Comment:
one thing that's potentially confusing is `example_inputs_` here is only going to not going to include non-Params.. will that work ?

Reply:
params come from `torch._guards.TracingContext.get().params_flat` which was added by your PR.

Reply:
Didn't you say `found the pure AOTInductor test (not cpp_wrapper tests) failed because inductor is called directly without going through dynamo and thus torch._guards.TracingContext`

so that won't always be set right ?